uaclient/exceptions.py

```diff
@@ -258,6 +258,20 @@ def __init__(self, msg: str, issue_id: str) -> None:
         )
 
 
+class GCPProAccountError(UserFacingError):
+    """An exception raised when GCP Pro service account is not enabled"""
+
+    def __init__(self, msg: str, msg_code: Optional[str], code=None):
+        self.code = code
+        self.error_msg = msg
+        super().__init__(msg, msg_code)
+
+    def __str__(self):
+        return "GCPProServiceAccount Error {code}: {msg}".format(
+            code=self.code, msg=self.error_msg
+        )
```

Comment:
I think `msg` is already stored in the parent class
```suggestion
    def __init__(self, msg: str, msg_code: Optional[str], code=None):
        self.code = code
        super().__init__(msg, msg_code)

    def __str__(self):
        return "GCPProServiceAccount Error {code}: {msg}".format(
            code=self.code, msg=self.msg
        )
```